tardis/transport/montecarlo/tests/conftest.py

```diff
@@ -138,3 +139,24 @@ def random_call():
         np.random.random()
 
     return njit(random_call)
+
+
+@pytest.fixture(scope="module")
+def simulation_last_interaction_tracking_enabled(
+    config_verysimple, atomic_dataset
+):
+    config_verysimple.montecarlo.iterations = 3
```

Comment:
I think we should have a new config with these values set that can be read in and used as is, rather than editing config_verysimple which might be used elsewhere.

Reply:
I have made a new configuration object and used it instead of editing the previous one.